src/main/java/de/nycode/bankobot/docdex/FlexmarkUtils.java

```diff
@@ -0,0 +1,121 @@
+// Credits: https://github.com/bankobotv14/BankoBot/blob/main/src/main/java/de/nycode/bankobot/docdex/FlexmarkUtils.java
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 BankoBot Contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+package de.nycode.bankobot.docdex;
```

Comment:
> Why is this not in the default package? 
Because that code was copied from another project to give them credit license and package remained similar to how it`s done with other projects.
>  And why Java?
Because writing flexmark code in kotlin is horrible